include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file jointMoteus.h
+ * @author J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @brief Moteus powered joint class
+ * @version 0.1
+ * @date 2022-02-22
+ * 
+ * @copyright BSD 3-Clause License, Copyright (c) 2021 The Trustees of the University of Pennsylvania. All Rights Reserved
+ * 
+ */
+
+#pragma once
+#include "kodlab_mjbots_sdk/jointBase.h"
+#include "kodlab_mjbots_sdk/moteus_protocol.h"
+
+namespace kodlab{
+/**
+ * @brief 
+ * Moteus Joint class that includes all the moteus specific vectors. 
+ */
+class JointMoteus: public JointBase{
+    public:
+        int can_bus;    /// the can bus the moteus communicates on
+        int can_id;     /// the can id of this joint's moteus
+
+        ::mjbots::moteus::Mode mode_ = ::mjbots::moteus::Mode::kStopped;    /// joint's moteus mode
+
+        /**
+         * @brief Construct a new Joint Moteus object
+         * 
+         * @param can_bus_      /// the can bus the moteus communicates on [1-4]
+         * @param can_id_       /// the can id of this joint's moteus [1-127]
+         * @param direction     /// 1 or -1, flips positive rotation direction (Default:1)
+         * @param zero_offset   /// offset [rad] of joint zero position from servo zero postition (Default:0)
+         * @param max_torque    /// Maximum torque of the joint [N m] (Default:inf)
+         * @param gear_ratio    /// Gear ratio joint to servo (ratio>1 means slower joint) (Default:1.0)
+         * @param pos_min       /// Minimum joint limit (Default:inf)
+         * @param pos_max       /// Maximum joint limit (Default:-inf)
+         */
+        JointMoteus(int can_bus_,int can_id_,
```

Comment:
Since the arguments are not member variables, they should not have an underscore after them